container/karaf/src/main/filtered-resources/etc/startup.properties

```diff
@@ -10,6 +10,8 @@ mvn\:org.osgi/org.osgi.util.promise/1.1.1 = 9
 mvn\:org.apache.felix/org.apache.felix.coordinator/1.0.2 = 9
 mvn\:org.apache.felix/org.apache.felix.converter/1.0.14 = 9
 mvn\:org.osgi/org.osgi.util.function/1.1.0 = 9
+# OPENNMS: Add part 1 of ConfigManager
+mvn\:org.opennms.features.config.osgi/org.opennms.features.config.osgi.del/30.0.0-SNAPSHOT = 9
```

Comment:
Use a placeholder for the version instead of making it fixed

Reply:
done.